src/chat/Chat.ts

```diff
@@ -50,10 +50,10 @@ export const createChatMessage = () => {
     })
 
     socket.on(Sockets.Message, (message: TextMessage) => {
-        const sendMsg = addMessage(message)
+        const sentMessage = addMessage(message)
 
         if (socket.id === message.userUUID) {
-            sendMsg.addClass('myMessage')
+            sentMessage.addClass('myMessage')
```

Comment:
`socket.id === message.userUUID`

This is missleading, `socket.id` is ID of the socket no id of the user. I would change it to `socket.roomID` or `socket.userUUID`

Reply:
why bad? userUUID is socket ID. when I later add the function private message I will refer to userUUID

Reply:
because you are an author of this code and you understand it, for me its not obvious that socket.id is userUUID